web-server/algorand_transaction/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod account;
```

Comment:
Same as the other case, we want to add `#![no_std]` to the top if the lib file here to build without linking to `std`.